runtime/src/lib.rs

```diff
@@ -125,7 +125,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     //   `spec_version`, and `authoring_version` are the same between Wasm and native.
     // This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
```

Comment:
`This value is set to 120...`

Reply:
I guess this comment could be wiped out since it is a template one.